paparazzi/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionTest.kt

```diff
@@ -30,7 +31,8 @@ class AccessibilityRenderExtensionTest {
     deviceConfig = DeviceConfig.NEXUS_5.copy(
       // Needed to render accessibility content next to main content.
       screenWidth = DeviceConfig.NEXUS_5.screenWidth * 2,
-      softButtons = false
+      softButtons = false,
+      orientation = ScreenOrientation.LANDSCAPE,
```

Comment:
I had to add this because the accessibility test doubles the screen width, which the new `DeviceConfig` would consider to be `LANDSCAPE`

Reply:
how come?  is screenWidth * 2 >= screenHeight?

Reply:
looks like the answer is yes...  my concern here is that this isn't straightforward to users.

@adamalyyan @geoff-powell can we move this double-width configuration out of the test and inside Paparazzi when it detects ARE is being used?